### ISSUE TITLE
Maybe have the calculator functionality split out into a separate repo, or at least separate ahk file in this one?

### ISSUE THREAD

**Issue by Balfa** [neon-dev/RDPHotkeyHelper#1]
"Also the menu key which usually opens the context menu is remapped to open the calculator instead"
lol, this little footnote at the bottom of the readme caused me a troubleshooting headache for quite a while.
I didn't put two and two together that pressing the context menu key opening calculator coincided with my installing of this script, and spent quite a while googling why my calculator was opening constantly 🤦

It's a bit of a random thing to have _included_ in this script. I understand this script is probably more of a personal helper script that you wrote for your own purposes, but as a "product" on github that you're pointing people too in forums, it might be worthwhile splitting these two unrelated functionalities into two separate scripts?

Overall, thanks for the script though. Been very useful otherwise 😁

**Comment by neon-dev** [comment]
Sorry. I just published what I use personally and didn't want to proactively invest the time to make a public script which I didn't know the requirements for.
A media key only script most likely would be sufficient, but I'm not sure if separate scripts for additional hotkeys would work well together, due to the way the hotkeys need to be (re)installed. So that's why I released everything as is and carefully named it HotkeyHelper and not MediaKeyHelper.
My preferred solution would be a config file with sensible defaults, maybe even with a UI or tray menu options. I currently don't have the time for it but would be happy to accept a PR.
